### ISSUE TITLE
Non-constant gas costs for token minting

### ISSUE THREAD

**Issue by GNSPS** [ConsenSys/curationmarkets#5]
https://github.com/ConsenSys/curationmarkets/blob/master/CurationMarkets.sol#L71

Implements a correct yet very expensive system for token minting that linearly increases in execution price with the token amount being bought.

I propose modeling this along the lines of what Bancor has done with their continuous price function.
(v. https://github.com/bancorprotocol/contracts/blob/master/solidity/contracts/BancorFormula.sol)
(v. https://drive.google.com/file/d/0B3HPNP-GDn7aRkVaV3dkVl9NS2M/view)

**Comment by simondlr** [comment]
Yeah. I suspected there's a way to improve it, but I couldn't math well enough to fit it into a specific function. Will definitely take a look at the Bancor Formula.
